### ISSUE TITLE
Accessibility: Define Unique and Relevant Page Titles

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/MIPs#208]
## Rationale

The page title is one of the first elements accessed by assistive technologies and it is also displayed in browser tabs and search engine results. A relevant and descriptive page title allows all users, including those with visual impairments, to quickly understand the purpose and content of the page. 

## 1. Functional Requirements

•	Each web page must have a <title> element in the <head> section of the HTML document.
•	The title must accurately describe the content or purpose of the page.
•	The title should be concise but informative enough to distinguish the page from others.
•	If the site uses templates (e.g., for blog posts or product pages), the title should include dynamic context-specific data (e.g., post title, product name).
•	Avoid generic titles like “Home”or “Page 1”.
•	If multiple pages share a similar layout, each must still have an **unique** and **contextually** meaningful title.
•	The page title should be the first thing announced by screen readers when a user navigates to a new page.

**NB:**

**The issue was partially resolved by [the ticket](https://builders.meeds.io/portal/mycraft/tasks/taskDetail/9029) , which introduced the following naming convention: [Page Name] – [Space Name or Site Name] – [Platform Name]. However, this convention does not fix the problem for the Notes pages, user profiles, events, projects, and tasks...**

### Feature Flags

| Property Name  | Default Value | Target Audience | Functional Behaviour |
| ------------- | ------------- | ------------- | ------------- |
|  |   |   |   |

### Access 
   - GUI 
   - API 

### Services & processing
   - Ex: Scheduled Jobs

### Data and persistence
   - Macro data model
   - Persistence layer 
   - Use of caches
## 5. Annexes
   - Reasoning that explain the technical choices
   - Impacts 
   - References
